aes/decryptor.go

```diff
@@ -19,6 +19,11 @@ type encryptedValue struct {
 
 const nonceSize int = 32
 
+type stashData struct {
+	iv        []byte
+	plaintext interface{}
+}
+
```

Comment:
This should be a public struct if it's being returned from a public function, `Decrypt`.